<ui version="4.0" >
 <class>MarbleNavigator</class>
 <widget class="QWidget" name="MarbleNavigator" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>5</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="6" column="1" >
      <widget class="QFrame" name="m_pSpacerFrame" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Plain</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QToolButton" name="moveUpButton" >
       <property name="toolTip" >
        <string>Up</string>
       </property>
       <property name="whatsThis" >
        <string>Tilts the globe's axis towards the user.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/arrow-up.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QToolButton" name="moveLeftButton" >
       <property name="toolTip" >
        <string>Left</string>
       </property>
       <property name="whatsThis" >
        <string>Rotates the globe counterclockwise around its axis.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/arrow-left.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Left</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="goHomeButton" >
       <property name="toolTip" >
        <string>Reset View</string>
       </property>
       <property name="whatsThis" >
        <string>Click this button to restore the original zoom level and view angle.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/go-home.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Home</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QToolButton" name="moveRightButton" >
       <property name="toolTip" >
        <string>Right</string>
       </property>
       <property name="whatsThis" >
        <string>Rotates the globe clockwise around its axis.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/arrow-right.png</iconset>
       </property>
       <property name="shortcut" >
        <string>Right</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QToolButton" name="moveDownButton" >
       <property name="toolTip" >
        <string>Down</string>
       </property>
       <property name="whatsThis" >
        <string>Tilts the globe's axis away from the user.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/arrow-down.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QToolButton" name="zoomInButton" >
       <property name="toolTip" >
        <string>Zoom In</string>
       </property>
       <property name="whatsThis" >
        <string>Zoom in by pressing this button to see more detail.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/zoom-in.png</iconset>
       </property>
       <property name="shortcut" >
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QSlider" name="zoomSlider" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>3</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Zoom Slider</string>
       </property>
       <property name="whatsThis" >
        <string>Use this slider to adjust the zoom level of the map.</string>
       </property>
       <property name="minimum" >
        <number>1000</number>
       </property>
       <property name="maximum" >
        <number>2500</number>
       </property>
       <property name="value" >
        <number>1050</number>
       </property>
       <property name="sliderPosition" >
        <number>1050</number>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1" >
      <widget class="QToolButton" name="zoomOutButton" >
       <property name="toolTip" >
        <string>Zoom Out</string>
       </property>
       <property name="whatsThis" >
        <string>Zoom out by pressing this button to see less detail.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
       <property name="icon" >
        <iconset resource="marble.qrc" >:/icons/zoom-out.png</iconset>
       </property>
       <property name="shortcut" >
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="marble.qrc" />
 </resources>
 <connections/>
</ui>
